docs/src/complete_sii.md

```diff
@@ -305,6 +313,164 @@ integrator.ps[:b] = 3.0
 setp(integrator, :b)(integrator, 3.0) # functionally the same as above
 ```
 
+## Parameter Timeseries
+
+If a solution object saves modified parameter values (such as through callbacks) during the
```

Comment:
```suggestion
If a solution object includes modified parameter values (such as through callbacks) during the
```

Reply:
Fixed in https://github.com/SciML/SymbolicIndexingInterface.jl/pull/75/commits/9046cd71a4147ac48322914436f422f847c4624f